src/crustls.h

```diff
@@ -268,6 +340,33 @@ void rustls_error(enum rustls_result result, char *buf, size_t len, size_t *out_
 
 bool rustls_result_is_cert_error(enum rustls_result result);
 
+/**
+ * Retrieve the nth element from the input slice of slices. If the input
+ * pointer is NULL, returns 0.
+ */
+size_t rustls_slice_slice_bytes_len(const struct rustls_slice_slice_bytes *input);
```

Comment:
Comment is copy+pasta. Been there, done that.

Reply:
This is odd, because the corresponding doccomment in `rslice.rs` is correct.